packages/server/postgres/queries/src/updateTeamByTeamIdQuery.sql

```diff
@@ -9,5 +10,7 @@ UPDATE "Team" SET
   "lastMeetingType" = COALESCE(:lastMeetingType, "lastMeetingType"),
   "tier" = COALESCE(:tier, "tier"),
   "orgId" = COALESCE(:orgId, "orgId"),
-  "updatedAt" = COALESCE(:updatedAt, "updatedAt")
-WHERE id = :id;
+  "updatedAt" = COALESCE(:updatedAt, "updatedAt"),
+  "eqChecked" = COALESCE(:eqChecked, "eqChecked")
+WHERE id IN :ids
+RETURNING *;
```

Comment:
I've always gone back & forth on returning the doc.
On the 1 hand, it's great because it's a single hit to the DB to get a write + a fresh read.
On the other hand, if I don't need that read, it's a little wasteful.

No changes needed, but if you come across some strong opinions either way would love to hear em!